js/chat.js

```diff
@@ -921,6 +997,16 @@ class UltralyticsChat {
     group.appendChild(actions);
   }
 
+  addUserMessageActions(group) {
+    if (group.querySelector(".ult-user-message-actions")) return;
+    const actions = this.el(
+      "div",
+      "ult-user-message-actions",
+      `<button class="ult-chat-send" data-action="edit" aria-label="Send" data-tooltip="Send"><span class="ult-icon-swap" data-icon="arrowUp">${this.icon("arrowUp")}</span></button>`,
```

Comment:
💭 **SUGGESTION**: The user message action button is labeled and tooled as "Send", but functionally it performs an "Edit and Restart" of the conversation. For accessibility and clarity, it would be better if the label/tooltip described the action more precisely (e.g. "Edit and restart"), especially for screen readers and users discovering the feature.

**Suggested change:**
```suggestion
      const actions = this.el("div","ult-user-message-actions",`<button class="ult-chat-send" data-action="edit" aria-label="Edit and restart" data-tooltip="Edit and restart"><span class="ult-icon-swap" data-icon="arrowUp">${this.icon("arrowUp")}</span></button>`);
```